src/app/data/services/http/http.service.ts

```diff
@@ -109,6 +113,50 @@ export class HttpService {
   getMaticPrices(): Observable<{ usd: number }> {
       return this.http.get<{ usd: number }>(`${this.localApi}/matic-info`);
   }
+  
+  getEthereumPools(): Observable<Pool[]> {
```

Comment:
http.service is basic implementation for http interractions
specific logic better to move to dedicated files

Reply:
fixed